### ISSUE TITLE
GNURoot Octave - "symbolic" package does not load correctly

### ISSUE THREAD

**Issue by yalla** [corbinlc/GNURootDebian#65]
Good Morning,

first: Awesome work! Just what I was looking for. I followed the instructions and installed everything, but I fail at using the "symbolic" package. Transscript from my session:

```
Sponsored by:

_____ _
|_ _| ___ ___ ___ _| | _ _ ___ ___
| | | -_|| _|| . || . || | || || -_|
|_| |___||_| |__,||___||_ ||_|_||___|
|___|

GNU Octave, version 3.8.2
Copyright (C) 2014 John W. Eaton and others.
This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details, type 'warranty'.

Octave was configured for "arm-unknown-linux-gnueabihf".

Additional information about Octave is available at http://www.octave.org.

Please contribute if you find this software useful.
For more information, visit http://www.octave.org/get-involved.html

Read http://www.octave.org/bugs.html to learn how to submit bug reports.
For information about changes from previous versions, type 'news'.

Your home directory can be found at /storage/emulated/0/GNURoot/home.
You can put .m files there or a .octaverc file there.
octave:1> pkg list
Package Name | Version | Installation directory
-------------------+---------+-----------------------
control *| 2.6.6 | .../packages/control-2.6.6
financial *| 0.4.0 | .../packages/financial-0.4.0
general *| 1.3.4 | .../packages/general-1.3.4
image *| 2.2.2 | .../packages/image-2.2.2
io *| 2.2.4 | .../octave/packages/io-2.2.4
missing-functions *| 1.0.2 | .../missing-functions-1.0.2
odepkg *| 0.8.4 | .../packages/odepkg-0.8.4
optim *| 1.4.0 | .../packages/optim-1.4.0
signal *| 1.3.0 | .../packages/signal-1.3.0
specfun *| 1.1.0 | .../packages/specfun-1.1.0
statistics *| 1.2.3 | .../packages/statistics-1.2.3 struct *| 1.0.10 | .../packages/struct-1.0.10
symbolic *| 1.1.0 | .../packages/symbolic-1.1.0
octave:2> pkg load symbolic
octave:3> syms x
error: 'syms' undefined near line 1 column 1
octave:3>
```
I'm currently trying to install the `python-sympy` package manually in a shell because I think that was a requirement for the `symbolic` package (it isn't currently installed), but maybe I'm doing something entirely wrong.

Any ideas?

**Comment by tomaklutfu** [comment]
Man, installing python-sympy requires almost 2gb space and downloads almost 1gb. Is there something wrong for my configuration?

**Comment by yalla** [comment]
Installing python-sympy didn't help.

**Comment by tomaklutfu** [comment]
I have a workaround. First, I did `pip install sympy`. Then, in octove prompt `pkg uninstall symbolic` followed by `pkg install -auto -forge symbolic`. `-auto` may not be needed.

**Comment by yalla** [comment]
Gee, I was lazy. I just saw your answer (as of right now, months later) and I'll try it out at once. My bad.

**Comment by yalla** [comment]
Alright, that was it! Since I just installed everything from scratch to be sure, I had to add a few extra steps:
1. install `python-pip`
2. install `liboctave-dev`

Then, as you described, unloaded symbolic, uninstalled, and installed. And you were right, the `-auto` parameter wasn't necessary.

Thanks for your help and sorry that it took so long to answer!

Then again, can't there anything been done to get the symbolic package working, which would be shipped by the installer?

Alex.
